corehq/apps/reports/templates/reports/project_health/partials/user_list.html

```diff
@@ -3,9 +3,13 @@
 <table class="table table-striped">
     <thead>
         <tr>
+            <th></th>
+            <th colspan="2">{% trans "Change in Forms Submitted" %}</th>
+        </tr>
```

Comment:
@benrudolph Yup, I think this was the layout that Sheel had in mind per the google doc: https://docs.google.com/document/d/1DrXuXIXVCDfvEaDRoBX6klct7FP9EaNRcyl3Bc6qeTc/edit#